### ISSUE TITLE
Ignoring paths does not work like that

### ISSUE THREAD

**Issue by func0der** [Elendev/nexus-composer-push#22]
https://github.com/Elendev/nexus-composer-push/blob/9dc7d473eb0f28adb5caf829575b90d3010a59af/src/ZipArchiver.php#L48-L50

These lines are not necessary and also wrong.

The Finder internally evaluates if we want to ignore VCS paths and adds the paths to ignore here: https://github.com/symfony/finder/blob/v4.4.4/Finder.php#L702

If you loop through the files, you are ignoring basically EVERYTHING and that by accident, too.
This is because you are passing a file object to the `Finder::notPath()` method, which gets converted to an array. The conversion works, but the output is not what `Finder::notPath()` excepts.

So basically you are ignoring random paths.

Just remove the loop completely and you should be good to go ;)
